.github/workflows/ci.yml

```
@@ -45,10 +45,24 @@ jobs:
           - '1.10'
         os:
           - ubuntu-latest
-          - macOS-latest
-          - windows-latest
         arch:
           - x64
+        trixi_test:
+          - tree_1d
+          - tree_2d
+          - structured_2d
+          - unstructured_2d
+          - unit
+          - upstream
+        include:
+          - version: '1.10'
+            os: macOS-latest
+            arch: x64
```

```suggestion
            os: macos-latest
            arch: aarch64
```
Make `os` consistent with the GitHub [docs](https://github.com/actions/runner-images?tab=readme-ov-file#available-images) and [fix](https://github.com/julia-actions/setup-julia?tab=readme-ov-file#inputs) `arch` for Apple Silicon.